unime/src/lib/credentials/Favorites.svelte

```diff
@@ -13,9 +11,12 @@
 
   export let credentialType: 'all' | 'data' | 'badges' = 'all';
 
-  let favorite_credentials: DisplayCredential[] = $state.credentials.filter((c) => c.metadata.is_favorite);
+  // Make favorite_credentials reactive in case we sort favorites in the future, too.
```

Comment:
Comment needs to be updated. After we made filtering of favorites responsive, they are sorted as well, because backend sorts all credentials.